packetery/src/Packetery/Module/Order/GridExtender.php

```diff
@@ -276,6 +276,7 @@ public function fillCustomOrderListColumns( string $column ): void {
 			return;
 		}
 
+		$wcOrderEntity = new Entity( $wcOrder );
```

Comment:
Zde to vyvolává dojem, že jde o WC_Order, použil bych $moduleOrder, jako jinde. Totéž v Synchronizeru.